config/software/bed_mesh.cfg

```diff
@@ -1,4 +1,8 @@
 [bed_mesh]
+# FOLLOW THE FORUMULA FOR relative_reference_index to be centered, for example:
```

Comment:
Can you also add a comment saying that this is the default config when calling `BED_MESH_CALIBRATE` without any parameters but it will be overiden on the fly if using the adaptive bed mesh macro ?

Reply:
Done